shapash/utils/check.py

```diff
@@ -4,13 +4,13 @@
 
```

Comment:
Maybe check_consistency_model_features start to be difficult to read between specific encoders and specif models